### PR TITLE
Fix ML Pipelines Tests

### PR DESCRIPTION
**Pull request by rimolive**
Fix the ML Pipelines CI Tests #653 

## Description
A change in how to check if the Pipeline Run exists after creation was added 

## How Has This Been Tested?
```
$ cd tests
$ make tests
$ make run TESTS_REGEX=ml-pipelines
```

## Merge criteria:

- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: tests/basictests/ml-pipelines.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ function list_pipelines() {
 function create_run() {
     header "Creating a run"
     RUN_ID=$(curl -s -H "Content-Type: application/json" -X POST ${ROUTE}/apis/v1beta1/runs -d "{\"name\":\"test-pipeline-run_run\", \"pipeline_spec\":{\"pipeline_id\":\"${PIPELINE_ID}\"}}" | jq -r .run.id)
-    os::cmd::try_until_text "curl -s ${ROUTE}/apis/v1beta1/runs/${PIPELINE_ID} | jq '.id'" "$RUN_ID" $odhdefaulttimeout $odhdefaultinterval
```
Comment (LaVLaS) [inline tests/basictests/ml-pipelines.sh:42]: You'll need to add update the `jq` statement to return `.run.id`

**Comment by LaVLaS** [comment]
@rimolive I ran another test on OCP 4.11 and it appears like the kfp pipeline api server is not deleting the `PipelineRun` object during the `delete_pipeline` function.  Can you confirm that it is passing on an OCP 4.10 cluster and the `PipelineRun` object is deleted after `curl -s -X DELETE ${ROUTE}/apis/v1beta1/pipelines/${PIPELINE_ID}` runs successfully?

**Review by LaVLaS — APPROVED** [review]
/lgtm

Manual execution on an OCP 4.11 cluster worked without any issues

**Comment by DharmitD** [comment]
Super trivial but should the `make tests` command in the instructions be `make build` instead?

**Pull request merged**